docs/how_to_guides/configuring_metadata_stores/how_to_configure_an_expectation_store_in_amazon_s3.rst

```diff
@@ -43,7 +54,13 @@ The configuration file tells Great Expectations to look for Expectations in a st
                     bucket: '<your_s3_bucket_name>'
                     prefix: '<your_s3_bucket_folder_name>'
 
-3. Confirm that the S3 Expectations store has been added by running ``great_expectations store list``. Notice the output contains two Expectation stores: the original ``expectations_store`` on the local filesystem and the ``expectations_S3_store`` we just configured.  This is ok, since Great Expectations will look for Expectations in the S3 bucket as long as we set the ``expectations_name`` variable to ``expectations_S3_store``.
+
+4. **Copy existing Expectation JSON files to the S3 bucket**. (This step is optional).
```

Comment:
Could this be done with `aws s3 sync`?

If the step is important enough to include, we should give users at least basic example instructions for how to accomplish it.